Detectors/ITSMFT/common/data/AlpideResponseData/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+# All rights not expressly granted are reserved.
+#
+# This software is distributed under the terms of the GNU General Public
+# License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+
+o2_add_executable(alpide-response-generator
+                  SOURCES AlpideResponse.cxx
+                  PUBLIC_LINK_LIBRARIES O2::ITSMFTSimulation
+                                        ROOT::Core
+                  TARGETVARNAME targetName)
+
+set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${CMAKE_SOURCE_DIR}/Detectors/ITSMFT/common/data/AlpideResponseData/AlpideResponse.cxx)
+
+add_custom_command(TARGET O2exe-alpide-response-generator POST_BUILD
+                   COMMAND ${CMAKE_BINARY_DIR}/stage/bin/o2-alpide-response-generator -i  $ENV{ITSRESPONSE_ROOT}/response/AlpideResponseData/ -o ${CMAKE_SOURCE_DIR}/Detectors/ITSMFT/common/data/AlpideResponseData/
```

Comment:
The ROOT file with responses should be installed under $O2_ROOT/share.  (Not under $CMAKE_SOURCE_DIR)

Reply:
> Thank you very much, @mconcas ! The generation of ROOT file with responses works. But :
> 
> 1. The file gets installed under the source directory, whereas the digitiser tries to get it from under the $O2_ROOT/share/.
> 2. Now, this PR needs to be rebased on dev.

Thanks, this I saw, and fixed in my local fork I was waiting to have the package in alidist.
I'll update the pr these days.